spec/spec_helper.rb

```diff
@@ -52,4 +52,13 @@ def ast(*args)
   config.run_all_when_everything_filtered = true
   config.include FactoryGirl::Syntax::Methods
   config.include Helpers
+
+  config.disable_monkey_patching!
+
+  config.mock_with :rspec do |mocks|
+    # Prevents you from mocking or stubbing a method that does not exist on
```

Comment:
I think the comment isn't needed – the reader can refer to the RSpec documentation if they're not familiar with this method.